api/services/FootprintService.js

```diff
@@ -181,16 +181,26 @@ module.exports = class FootprintService extends Service {
    * @return Promise
    */
   createAssociation(parentModelName, parentId, childAttributeName, values, options) {
-    /*
-     const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
-     const childAttribute = parentModel.associations[childAttributeName]
-     const childModelName = childAttribute.target.name
-
-     values[childAttribute.foreignKey] = parentId
+    const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
```

Comment:
Sounds good :) I thought I remembered there being an error class for that, but I forgot what it was. I've implemented that and am pushing now.